src/test/javascript/spec/component/programming-exercise/programming-exercise-custom-build-plan.component.spec.ts

```diff
@@ -92,9 +84,8 @@ describe('ProgrammingExercise Custom Build Plan', () => {
     });
 
     it('should accept editor', () => {
-        const elementRef: ElementRef = new ElementRef(document.createElement('div'));
         expect(comp.editor).toBeUndefined();
-        comp.editor = new MonacoEditorComponent(mockThemeService, elementRef, renderer2, translateService);
+        comp.editor = TestBed.createComponent(MonacoEditorComponent).componentInstance;
         expect(comp.editor).toBeDefined();
```

Comment:
If this looks odd to you: `comp.editor` is actually a `get`/`set` pair. We will remove this when refactoring the build plan components